Adding AI builders to getting started with js tracking snippet (#10763)
PostHog/posthog.com: contents/docs/integrate/_snippets/install-js-snippet.mdx

  If you need ES5 support for example to track Internet Explorer 11 replace `/static/array.js` in the snippet with `/static/array.full.es5.js`
</details>

<details>

>   <summary>Working with AI builders?</summary>

Reviewer: ```suggestion
  <summary>Working with AI code editors?</summary>
```

>   If you’re working with AI builders (like Loveable, Bolt.new, Replit and others), it’s easy to introduce PostHog tracking. All you have to do is ask. (Seriously.) Specifically: in your tool, type a prompt like this: “I’d like to use PostHog to track events here; here are my install snippets.” Paste in your snippets. The tool will do the rest.

Reviewer: ```suggestion
  If you’re working with AI builders (like Loveable, Bolt.new, Replit, and others), it’s easy to install PostHog. All you have to do is ask (seriously). Specifically: in your tool, type a prompt like this: “I’d like to use PostHog to track events here; here is my install snippet.” Paste in your snippet. The tool will do the rest.
```

> As of this time, some web-based AI builders (such as artefacts by Claude) don’t support access to external JS libraries. You’ll be able to ask it to use a placeholder, and then update it manually when you deploy to production.

Reviewer: ```suggestion
As of this time, some web-based AI builders (such as artifacts by Claude) don’t support access to external JS libraries. You’ll be able to ask it to use a placeholder, and then update it manually when you deploy to production.
```
